sdk/payload.mk

```diff
@@ -1,7 +1,7 @@
 #---------------------------------------------------------------------------------
 # options for code generation
 #---------------------------------------------------------------------------------
-CFLAGS	+= $(INCLUDE) -fno-asynchronous-unwind-tables -fno-builtin
+CFLAGS	+= $(INCLUDE) -fno-asynchronous-unwind-tables -fno-builtin -fpic
```

Comment:
i'd need to test, but i think the no-builtin can be removed, they can be useful stuff tbh